plugin/trino-hive/src/test/java/io/trino/plugin/hive/orc/TestHiveOrcWithShortZoneId.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.orc;
+
+import io.trino.plugin.hive.HiveQueryRunner;
+import io.trino.testing.AbstractTestQueryFramework;
+import io.trino.testing.QueryRunner;
+import org.testng.annotations.Test;
+
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static io.trino.testing.containers.TestContainers.getPathFromClassPathResource;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class TestHiveOrcWithShortZoneId
+        extends AbstractTestQueryFramework
+{
+    private String resourceLocation;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        // See README.md to know how resource is generated
+        resourceLocation = getPathFromClassPathResource("with_short_zone_id/data");
+        return HiveQueryRunner.builder()
+                .build();
+    }
+
+    @Test
+    public void testSelectWithShortZoneIdDisabled()
+    {
+        String tableName = "test_select_with_short_zone_id_disabled_" + randomNameSuffix();
+
+        assertUpdate("CREATE TABLE %s (id INT, firstName VARCHAR, lastName VARCHAR) WITH (external_location = '%s')".formatted(tableName, resourceLocation));
```

Comment:
Can we use `TestTable` here which would take care of random tableNames and dropping them post running the tests.